### DISCUSSION TITLE
[Feature] Support for feature flag env

### DISCUSSION THREAD

**Discussion by webysther** [immich-app/immich#1700] [Feature Request]
### The feature

Support for enable or disable feature using a flag (env vars) will be great, with this is possible to customize the server, in case someone don't need video just turn off or object detection or disable the theme selection, albums or sharing.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
You can now configure features via the config file https://immich.app/docs/install/config-file
